server/ladder_service/ladder_service.py

```diff
@@ -565,32 +674,6 @@ async def shutdown(self):
             queue.shutdown()
 
 
-def game_name(*teams: list[Player]) -> str:
-    """
-    Generate a game name based on the players.
-    """
-
-    return " Vs ".join(_team_name(team) for team in teams)
-
-
-def _team_name(team: list[Player]) -> str:
-    """
-    Generate a team name based on the players. If all players are in the
-    same clan, use their clan tag, otherwise use the name of the first
-    player.
-    """
-    assert team
-
-    player_1_name = team[0].login
-
-    if len(team) == 1:
-        return player_1_name
-
-    clans = {player.clan for player in team}
-
-    if len(clans) == 1:
-        name = clans.pop() or player_1_name
-    else:
-        name = player_1_name
-
-    return f"Team {name}"
+class NotConnectedError(asyncio.TimeoutError):
```

Comment:
I think GameClosedError and NotConnectedError should be defined at the same place.

Reply:
Hmm, GameClosedError is only raised from withing LadderGame, but NotConnectedError is only raised from LadderService. So IDK where a better place to put them would be.